pwa-devdocs/src/_includes/layout/pre-release-banner.html

```diff
@@ -1,4 +1,6 @@
-<div class="message-banner">
-If you have any project questions, concerns, or contribution ideas, join our <a href="https://magentocommeng.slack.com/messages/C71HNKYS2"><strong>#pwa</strong></a> Slack channel.
-Find out how to create an account by visiting <a href="https://devdocs.magento.com/community/resources/resources.html">Community Resources</a>
+<div class="message-banner warning-banner">
+    The <a href="https://github.com/magento-research/pwa-studio">PWA Studio repository</a> is moving to a new GitHub organization!
+    As a result, the URL for this site is also changing to <a href="https://magento.github.io/pwa-studio/">https://magento.github.io/pwa-studio/</a>.
```

Comment:
```suggestion
    As a result, the URL for this site is also changing. Please use <a href="https://pwastudio.io">https://pwastudio.io"</a> going forward.
```

Let's point folks to pwastudio.io as much as possible. In case we ever change locations again, we can just update the redirect.